docs/tutorial/spellchecker.md

```diff
@@ -0,0 +1,74 @@
+# SpellChecker
+
+Electron has built-in support for Chromium's spellchecker since Electron 8.  On Windows and Linux this is powered by Hunspell dictionaries, on macOS it makes use of the native spellchecker APIs.
```

Comment:
nit(comma splice): On Windows and Linux this is powered by Hunspell dictionaries, **and** on macOS it makes use of the native spellchecker APIs.